activiti-cloud-services-org/activiti-cloud-services-org-rest/pom.xml

```diff
@@ -51,6 +51,10 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-annotations</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-validation</artifactId>    
```

Comment:
@igdianov, Is it ok to have this starter here?

Reply:
I think it is OK, but I will double check

Reply:
@erdemedeiros I have removed `spring-boot-starter-validation` from compile scope